src/vmm_core.ml

```diff
@@ -99,19 +160,10 @@ module Name = struct
 
   let name (_, name) = name
 
-  let create path name =
-    if valid_label name then
-      Ok (path, Some name)
-    else
-      Error (`Msg ("invalid name: " ^ name))
+  let create path name = path, Some name
 
   let create_of_path p = (p, None)
 
-  let create_exn path name =
-    match create path name with
-    | Ok t -> t
-    | Error `Msg m -> invalid_arg m
-
   let drop_path (_, name) = [], name
 
   let rec drop_prefix_exn (p, name) p' =
```

Comment:
I see this function wasn't touched, but I notice it uses `String.equal` instead of `Label.equal` and that's likely too strict.